src/features/click-to-load.js

```diff
@@ -749,6 +792,7 @@ function getLearnMoreLink (mode = 'lightMode') {
     linkElement.href = 'https://help.duckduckgo.com/duckduckgo-help-pages/privacy/embedded-content-protection/'
     linkElement.target = '_blank'
     linkElement.textContent = sharedStrings.learnMore
+    linkElement.classList.add('learnMoreLink')
```

Comment:
Any particular motivations for doing a class here and not an id like in the other elements?

Reply:
> I wonder if it would be best to create a new text in the config instead of moving the "learn more" around.

Yea, you're right there. Moving the string around like this isn't correct really, and with the new preview sub-heading string the link is included. I figured this is pragmatic for now though, in particular since we're doing the same thing when adding this particular link - appending it to a div with the text content of `hoverTextBody + ' '`.